generators/server/templates/src/main/java/package/config/CacheConfiguration.java.ejs

```diff
@@ -24,12 +24,6 @@ import java.time.Duration;
 import org.ehcache.config.builders.*;
 import org.ehcache.jsr107.Eh107Configuration;
 
-    <%_ if (enableHibernateCache) { _%>
```

Comment:
From what I understand this block is part of the conditional block `if (cacheProvider === 'ehcache')`, and the other is part of the conditional block `if (cacheProvider === 'caffeine')`, so the imports wont be duplicated and if you use ehcache as cache provider, these imports will be missing.

Reply:
oh my bad, you're right.
So nothing to do here :D, I close